docs/chapters/2-requirements-elicitation.md

```diff
@@ -45,3 +45,6 @@ Se formularon las siguientes preguntas que nos ayudarán a conocer sobre las rut
 #### Empathy Mapping
 
 #### As-is Scenario Mapping
+<div class="as-is-map-img-container">
+  <img src="../static/as-is-scenario-mapping.png" alt="As-is Scenario Mapping" title="As-is Scenario Mapping" />
+</div>
```

Comment:
You can use the [Markdown image format](https://www.markdownguide.org/basic-syntax/#images-1) in Markdown documents instead of directly using HTML.

```suggestion
![As-is Scenario Mapping](../static/as-is-scenario-mapping.png)
```